packages/perf/android/src/main/java/io/invertase/firebase/perf/ScreenTrace.java

```diff
@@ -0,0 +1,227 @@
+package io.invertase.firebase.perf;
+
+/**
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import android.app.Activity;
+import android.util.Log;
+import android.util.SparseIntArray;
+import android.view.WindowManager;
+
+import androidx.core.app.FrameMetricsAggregator;
+
+import com.google.firebase.perf.FirebasePerformance;
+import com.google.firebase.perf.metrics.Trace;
+import com.google.firebase.perf.util.Constants;
+
+/**
+ * Utility class to capture Screen rendering information (Slow/Frozen frames) for the
+ * {@code Activity} passed to the constructor {@link ScreenTrace#ScreenTrace(Activity, String)}.
+ * <p>
+ * Learn more at https://firebase.google.com/docs/perf-mon/screen-traces?platform=android.
+ * <p>
+ * A slow screen rendering often leads to a UI Jank which creates a bad user experience. Below are
+ * some tips and references to understand and fix common UI Jank issues:
+ * - https://developer.android.com/topic/performance/vitals/render.html#fixing_jank
+ * - https://youtu.be/CaMTIgxCSqU (Why 60fps?)
+ * - https://youtu.be/HXQhu6qfTVU (Rendering Performance)
+ * - https://youtu.be/1iaHxmfZGGc (Understanding VSYNC)
+ * - https://www.youtube.com/playlist?list=PLOU2XLYxmsIKEOXh5TwZEv89aofHzNCiu (Android Performance Patterns)
+ * <p>
+ * References:
+ * - Fireperf Source Code: https://bityl.co/5v2O
```

Comment:
```suggestion
 * - Fireperf Source Code: https://bityl.co/5v2O
```

This link is `bityl` instead of bitly, but in general it's best to put a real code reference in there. Branch names change, code lines move around, but a permalink to the actual code is best I think as opposed to any shortening. When you can see the actual link structure it is usually possible to find the current link even if linkrot takes hold

Same applies to all similar links through this file

Reply:
fixed